src/renderer/type/js-doc-renderer.ts

```diff
@@ -98,19 +99,28 @@ export const defaultJsDocRenderOptions: Required<JSDocRenderOptions> = {
     };
   },
 
-  renderFieldDocs: ({ field }) => {
+  renderFieldDocs: ({ field, control }) => {
+    const tags: OptionalKind<JSDocTagStructure>[] = [
+      {
+        tagName: 'name',
+        text: field.name,
+      },
+      {
+        tagName: 'localized',
+        text: field.localized.toString(),
+      },
+    ];
+
+    if (control?.settings?.helpText) {
```

Comment:
While semantically not 100% correct, we could use the [@summary](https://jsdoc.app/tags-summary.html) tag for it - wdyt?

Reply:
Changed to `@summary` tag.